transport/src/commonTest/kotlin/com/genesys/cloud/messenger/transport/network/WebMessagingApiTest.kt

```diff
@@ -43,6 +48,27 @@ import kotlin.test.assertTrue
 class WebMessagingApiTest {
     private lateinit var subject: WebMessagingApi
 
+    private fun clientWith(
```

Comment:
We have mockHttpClientWith that does something similar. You can reuse it, just  parameterize where needed to enable/disable retries.